src/components/accounts/AccountFormAccountId.js

```diff
@@ -133,16 +133,21 @@ class AccountFormAccountId extends Component {
 
     handleCheckAvailability = (accountId, type) => (
         accountId
-            && !(
-                type === 'create' 
-                && (!this.handleAccountIdLengthState(accountId) 
-                && !this.checkAccountIdLength(accountId))
+            && this.isImplicitAccount()
```

Comment:
feels very weird that method takes `accountId` and then we only check `this.state.accountId` here